generate_analysis_graphs.py

```diff
@@ -6,6 +6,7 @@
 from glob import glob
 
 import altair
```

Comment:
Is this still used?

Reply:
Yes, because this only a partial migration. It's removed completely in #30